platform/self_hosting/getting_started.mdx

```diff
@@ -0,0 +1,20 @@
+---
+id: getting_started
+title: Getting Started
+sidebar_label: Getting Started
+---
+
+It is great to have you here! This guide will help you get started with the self-hosted version of Tolgee!
+
+## Installation
+First, you need to decide whether you want to run Tolgee with Docker or Java.
+
+The easiest way to run Tolgee is with **Docker**. You can find more information about Docker installation [here](running_with_docker).
+
+If you want to run Tolgee with **Java**, you can find more information about Java installation [here](running_with_java).
+
+## Configuration
+After you have installed Tolgee, you need to configure it. You can view the configuration guide for the self-hosted version [here](configuration).
+
+## Pricing
+Pricing for the self-hosted version is different from the cloud version. You can find the pricing [here](/pricing/self-hosted).
```

Comment:
I'd add this part once self-hosting pricing is released. Also, I'd describe more details about free self-hosting and paid version